strongbox-data-service/src/main/java/org/carlspring/strongbox/data/server/EmbeddedOrientDbServer.java

```diff
@@ -55,18 +73,117 @@ public void start()
         }
     }
 
+    public File getStudioClasspathLocation()
+    {
+        return new File(OServer.class.getProtectionDomain()
+                                     .getCodeSource()
+                                     .getLocation()
+                                     .getPath()).toPath().resolveSibling("orientdb-studio-2.2.0.jar").toFile();
```

Comment:
This code would have to keep being updated every time there is a newer version of the library. Would it be possible to improve this so that it could be a bit more dynamic and future-proof?

Reply:
The thing is that we need to know OrientDB jar file name and this was the only way I found. Also I made a constant with the version for it to be more convenience to update versions.